sysutils/aws-iam-authenticator/Portfile

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           golang 1.0
+
+# UPDATE THESE IN SYNC!
+set version         0.5.2
+set commit          292b9b82df69b87af962b92485b254d9f4b10f00
+
+go.setup            github.com/kubernetes-sigs/aws-iam-authenticator ${version} v
+revision            0
+go.package          sigs.k8s.io/aws-iam-authenticator
+
+description         AWS IAM Authenticator for Kubernetes
+long_description    Use IAM credentials to authenticate to Kubernetes.
+
+categories          sysutils
+platforms           darwin
+license             Apache-2
+
+maintainers
```

Comment:
`maintainers` must be set.  Refer to:  https://guide.macports.org/#reference.keywords.maintainers

For example:
```
maintainers @asobrien \
            openmaintainer
```

Reply:
👍  Thanks! This is my first one, added in f36cb24d5af

Reply:
OK great.  Also, if you're open to it, it would be good to add your email as well.  This way, the MacPorts buildbots can email you if your port fails to build there (the buildbots build your port for each version of macOS after your port has been committed to the tree).

Example:
```
maintainers         {gmail.com:aso.brien @asobrien} \
                    openmaintainer
```